api/prometheus/v1/api_test.go

```diff
@@ -40,10 +40,8 @@ type apiTest struct {
 	inRes        interface{}
 
 	reqPath   string
-	reqParam  url.Values
```

Comment:
They are entirely unused, just like `warnings` was. It seemed there were other mechanisms in place to construct the request params (using the `do` field in each testcase)

Reply:
But I was also surprised that this amount of dead code would be here, so maybe the linter and me misinterpreted something? :)

Reply:
We might have changed the comparison method or something similar without realizing their intended use. Let's make sure if we can actually use them. Otherwise, happy to accept clean-ups.

Reply:
So from what I can see, the `apiTest` structure was introduced via #165 (released in v0.8.0). However I could not find a single revision in this repo that actually used the `reqParam` field.

Reply:
That's an ancient PR :sweat_smile: Thanks for digging it up. Then I'm ok with the changes. If anything becomes broken, we can fix them on consecutive PRs.